lib/import/colorectal/providers/oxford/oxford_handler_colorectal.rb

```diff
@@ -22,6 +24,35 @@ def process_fields(record)
             res.each { |cur_genotype| @persister.integrate_and_store(cur_genotype) }
           end
 
+          def colorectal_file?
+            file_name = @batch.original_filename
+            file_path = File.dirname(file_name)
+            file_path_array = file_name.split('/')
+            pseudo_file = file_path_array[file_path_array.length - 1]
+            pseudo_filename = pseudo_file.sub(/.xls[x]?.pseudo/, '')
+            directory = Rails.root.join("private/pseudonymised_data/#{file_path}").to_s
+            csv_files = Dir.glob("#{directory}/*#{pseudo_filename}*_pretty.csv")
+
+            raise "Pretty CSV file not able to map for #{pseudo_file}" if csv_files.empty?
+
+            csv = CSV.read(csv_files[0], headers: true)
+            brca1_count, apc_count, mlh1_count = get_csv_counts(csv)
+
+            mlh1_count + apc_count > brca1_count
+          end
+
+          def get_csv_counts(csv)
+            brca1_count = csv['mapped:gene'].tally['7'].to_i
+            apc_count = csv['mapped:gene'].tally['358'].to_i
+            mlh1_count = csv['mapped:gene'].tally['2744'].to_i
+            [brca1_count, apc_count, mlh1_count]
```

Comment:
Similar to [comment](https://github.com/NHSDigital/data_management_system/pull/56/files#r1470077123) above

Reply:
See commit [fc77d32](https://github.com/NHSDigital/data_management_system/pull/56/commits/fc77d32297750f5ad5e7af05a8448808d070bffa)